app-misc/flirc/flirc-3.22.2.ebuild

```diff
@@ -3,17 +3,17 @@
 
 EAPI=6
 
-inherit desktop udev unpacker
+inherit desktop udev
 
 DESCRIPTION="Allows you to pair any remote control with your computer or media center"
 HOMEPAGE="https://flirc.tv/"
 SRC_URI="
-	amd64?	( https://packagecloud.io/Flirc/repo/packages/ubuntu/artful/flirc_${PV}_amd64.deb/download.deb  -> ${P}_amd64.deb )
-	arm?	( https://packagecloud.io/Flirc/repo/packages/debian/stretch/flirc_${PV}_armhf.deb/download.deb -> ${P}_armhf.deb )"
+	amd64?	( http://apt.flirc.tv/arch/x86_64/flirc.latest.x86_64.tar.gz -> ${P}_amd64.tar.gz )
```

Comment:
Any chance you could find versioned URLs somewhere? This is going to suck hard given it's mirror-restricted.